alacritty.yml

```diff
@@ -197,9 +197,21 @@ background_opacity: 1.0
 mouse_bindings:
   - { mouse: Middle, action: PasteSelection }
 
+# Mouse settings
+#
+# The `faux_scrollback_lines` setting controls the number
```

Comment:
Can these comments be attached to each property instead? Would allow them to be a bit shorter and directly associated with the property the document.